### PR TITLE
feat: add project commands skeleton

### PR DESCRIPTION
**Pull request by noetarbouriech**
## Changes

- Add `paastech init` command skeleton. 
- Add `paastech delete` command skeleton.
- Add `paastech env [add|rm|ls]` command skeleton.
- Add `paastech dashboard` command.

## Notes

Most commands are not implemented yet as they require the API to be operational.

### CODE REVIEW

File: cmd/dashboard.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+	"github.com/spf13/cobra"
+)
+
+var dashboardCmd = &cobra.Command{
+	Use:   "dashboard",
+	Short: "Open dashboard on web UI",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// only support linux for now
+		return exec.Command("xdg-open", "https://paastech.cloud/#/dashboard").Start()
```
Comment (mfernd) [inline cmd/dashboard.go:14]: maybe this?
[github.com/pkg/browser](https://github.com/pkg/browser)

File: cmd/env.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	"github.com/spf13/cobra"
+)
+
+var envCmd = &cobra.Command{
+	GroupID: "project",
+	Use:     "env",
+	Short:   "Manage environment variables for project",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return errors.New("not enough arguments")
```
Comment (mfernd) [inline cmd/env.go:17]: capital letter here for the error or the opposite

File: cmd/root.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ var rootCmd = &cobra.Command{
 🍉 PaaSTech CLI is the official tool to manage and operate your 
    applications and deployments on the PaaSTech platform.
 
-🔗 Find more informations at: TBA
+🔗 Find more informations at: https://paastech.cloud/
```
Comment (mfernd) [inline cmd/root.go]: no `s` to information

**Review by mfernd — CHANGES_REQUESTED** [review]

File: cmd/dashboard.go (hunk 1 above)
Comment (noetarbouriech) [reply cmd/dashboard.go:14]: No need for another external dependency for such a trivial task.
xdg is enough for linux and windows and macos support will require more in-depth refactoring

**Review by mfernd — APPROVED** [review]

**Pull request merged**